.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
```

Comment:
@coderabbitai provide me the list of all merged PRs for milestone 1.19.0 that has a bug label. Give me the exhaustive list.

Reply:
Give me the exhaustive list.

Reply:
Give me the exhaustive list

Reply:
There should be more, try again

Reply:
Should be more than 30

Reply:
This is correct, but why this time the correct result is returned

Reply:
What should I say next time to get this correct result